api/src/MemberMatch/Members.cs

```diff
@@ -0,0 +1,105 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace RaceResults.MemberMatch
+{
+    public class Members
+    {
+        private Dictionary<string, HashSet<Member>> nameToMemberSet;
+        private HashSet<string> citySet;
+
+        public Members(string filename)
+        {
+            this.nameToMemberSet = new Dictionary<string, HashSet<Member>>();
+            this.citySet = new HashSet<string>();
+
+            foreach (string line in File.ReadLines(filename).Skip(1))
+            {
+                var fields = line.Split('\t');
+                Trace.Assert(fields.Length == 4, "Expect four fields in the 'sample_member.tsv' file");
```

Comment:
Is this class expected to only be used in the context of your test?

Reply:
Yes and no. I expect these to evolve as we connect with the rest of the system (for example, by adding GUIDs). On the other hand, should still be able to run these tests on a local machine without and DB connection.)